src/components/InfoViewer/InfoViewer.tsx

```diff
@@ -11,46 +14,89 @@ export interface InfoViewerItem {
     value: React.ReactNode;
 }
 
+export interface InfoViewerHeaderLabel {
+    value: React.ReactNode;
+    icon?: IconData;
+    theme?: LabelProps['theme'];
+}
+
 export interface InfoViewerProps {
     title?: React.ReactNode;
+    titleSuffix?: React.ReactNode;
+    separator?: React.ReactNode;
     info?: InfoViewerItem[];
     dots?: boolean;
     size?: 's';
     variant?: 'default' | 'small';
     className?: string;
     multilineLabels?: boolean;
     renderEmptyState?: (props?: Pick<InfoViewerProps, 'title' | 'size'>) => React.ReactNode;
+    headerLabels?: InfoViewerHeaderLabel[];
 }
 
 const b = cn('info-viewer');
 
 export const InfoViewer = ({
     title,
+    titleSuffix,
+    separator = '•',
     info,
     dots = true,
     size,
     variant = 'default',
     className,
     multilineLabels,
     renderEmptyState,
+    headerLabels,
 }: InfoViewerProps) => {
     if ((!info || !info.length) && renderEmptyState) {
         return <React.Fragment>{renderEmptyState({title, size})}</React.Fragment>;
     }
 
     return (
         <div className={b({size, variant}, className)}>
-            {title && <div className={b('title')}>{title}</div>}
+            <Flex
+                className={b('header')}
+                justifyContent="space-between"
+                gap={2}
+                alignItems="center"
+            >
+                {title && (
+                    <Flex gap="1" alignItems="center">
+                        <div className={b('title')}>{title}</div>
+                        {titleSuffix && (
+                            <React.Fragment>
+                                <div className={b('title-suffix')}>{separator}</div>
+                                <div className={b('title-suffix')}>{titleSuffix}</div>
+                            </React.Fragment>
+                        )}
+                    </Flex>
+                )}
+                {headerLabels && headerLabels.length > 0 && (
+                    <Flex gap={1} alignItems="center">
+                        {headerLabels.map((label, index) => (
```

Comment:
it's not a good practice to use index as key for element (in some cases it's appropriate, but I would avoid it as possible). May we use anything else as key? https://react.dev/learn/rendering-lists#why-does-react-need-keys